app/components/katalyst/table_component.rb

```diff
@@ -37,6 +38,7 @@ def initialize(collection:,
                    sorting: nil,
                    header: true,
                    caption: true,
+                   generate_ids: true,
```

Comment:
I don't want this to be on by default – my philosophy with tables is that they should generate absolutely minimal syntax. I think that adding IDs is a niche use-case – as we saw it's not even required for morphing.